planner/aamwriter/src/main/resources/examples/drafts/simple_adp.yaml

```diff
@@ -0,0 +1,104 @@
+tosca_definitions_version: tosca_simple_yaml_1_0_0_wd03
+
+description: AAM for SeaClouds Demo V2.0
+
+imports:
+    - tosca-normative-types:1.0.0.wd03-SNAPSHOT
+
+topology_template:
+    node_templates:
+        www:
+            type: sc_req.www
+            properties: {}
+            artifacts: { war: www.war, type: tosca.artifacts.File }
+            requirements:
+                - host: aws-ec2:us-west-2
+                
+        db:
+            type: sc_req.db
+            properties:
+                username: dbuser
+                password: dbpassword
+            requirements:
+                - host: pivotal
+                
+        aws-ec2:us-west-2:
+            type: seaclouds.nodes.Compute.Amazon
+            properties:
+              type: compute

```

Comment:
All the files in examples/drafts are drafts and should not taken into account. I will remove them.